### PR TITLE
[Cards] Added proper readme with overview, installation, and usage.

### PR DESCRIPTION
**Pull request by yarneo**
Also added a jazzy generated yaml so API can be generated.

### CODE REVIEW

File: components/Cards/README.md [hunk 1]
```diff
@@ -14,28 +14,79 @@ api_doc_root: true
...
 <ul class="icon-list">
-  <li class="icon-list-item icon-list-item--spec"><a href="https://material.io/guidelines/components/cards.html">Material Design guidelines: Cards</a></li>
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/cards.htm">Material Design guidelines: Cards</a>
+  </li>
 </ul>
 
+## Installation
+
+### Requirements
```
Comment (ianegordon) [inline components/Cards/README.md]: We are not putting requirements in individual components any more.  
They have been moved to the main README.md.
We can remove the ### Requirements section.

File: components/Cards/README.md [hunk 2]
```diff
@@ -14,28 +14,79 @@ api_doc_root: true
...
+
+``` bash
+pod install
+```
+
+- - -
+
+### Overview
+
+Cards can be used either as a standalone `UIControl`, or as a `UICollectionViewCell`.
```
Comment (ianegordon) [inline components/Cards/README.md]: Consider:
Cards provides two different versions, MDCCard inheriting from UIControl and MDCCollectionViewCell inheriting from UICollectionCell.

File: components/Cards/README.md [hunk 3]
```diff
@@ -14,28 +14,79 @@ api_doc_root: true
...
+
+A card consists of different visual styling based on the current state it is in.
```
Comment (ianegordon) [inline components/Cards/README.md]: A card's state determines its visual styling.

File: components/Cards/README.md [hunk 4]
```diff
@@ -58,6 +109,10 @@ pod install
 
 ## Usage
 
+An `MDCCard` can be added and used as you would add any `UIView` or `UIControl`, if manually in code, or through the interface builder.
+
+An `MDCCardCollectionCell` can be added, used, and reused as a `UICollectionViewCell`, if manually in code, or through the interface builder.
+
 ### Importing
 
 Before using Cards, you'll need to import it:
```
Comment (ianegordon) [inline components/Cards/README.md:112]: #### Swift
``` swift
import MaterialComponents.MaterialCards
```


#### Objective-C
``` objc
#import "MaterialCards.h"
```

**Review by ianegordon — CHANGES_REQUESTED** [review]

File: components/Cards/README.md (hunk 4 above)
Comment (yarneo) [reply components/Cards/README.md:112]: I'm not sure what to do here; this looks like a copy of what we already have?

Comment (ianegordon) [reply components/Cards/README.md]: I don't see the actual import code sample in the existing PR.
The Swift & ObjC sample code would be consistent with our other readmes:
See Usage - Importing in:
https://github.com/material-components/material-components-ios/blob/develop/components/FeatureHighlight/README.md

Comment (ianegordon) [reply components/Cards/README.md:112]: It was collapsed.  Disregard.

**Review by ianegordon — APPROVED** [review]

**Pull request merged**